src/aat/java/uk/gov/hmcts/reform/em/orchestrator/performance/NpaLoad.scala

```diff
@@ -15,7 +15,7 @@ class NpaLoad extends Simulation with HttpConfiguration {
 //    var jsonObject = new JSONObject().put("bundle", testUtil.getDocumentId).toString()
 //    def run = exec(
 //          http("create-npa-task-" + testConfig.pdfName)
-//            .post(Env.getTestUrl + "/api/document-tasks")
+//            .startStitchingTask(Env.getTestUrl + "/api/document-tasks")
```

Comment:
We should kill all the old scala load tests - I've made a task for it: https://tools.hmcts.net/jira/browse/EM-2116